### PR TITLE
Release/v7.4 package references

### PR DESCRIPTION
**Pull request by TravisEz13**
# PR Summary

## PR Checklist

- [ ] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
  - Use the present tense and imperative mood when describing your changes
- [ ] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
  - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
  - [ ] None
  - **OR**
  - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/main/reference/7.5/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
    - [ ] Experimental feature name(s): 
- **User-facing changes**
  - [ ] Not Applicable
  - **OR**
  - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
  - [ ] N/A or can only be tested interactively
  - **OR**
  - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
  - [ ] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
  - **OR**
  - [ ] I have considered the user experience from a tooling perspective and opened an issue in the relevant tool repository. This may include:
    - [ ] Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode
        (which runs in a different PS Host).
      - [ ] Issue filed: 
    - [ ] Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
      - [ ] Issue filed: 
    - [ ] Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
      - [ ] Issue filed: 
    - [ ] Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).
      - [ ] Issue filed:

### CODE REVIEW

File: build.psm1 [hunk 1]
```diff
@@ -1450,8 +1451,9 @@ function Start-PSPester {
 
     # All concatenated commands/arguments are suffixed with the delimiter (space)
 
+    $command = "Write-Verbose `"PowerShell Version under test: `$($PSVersionTable.PSVersion)`" -Verbose;"
```
Comment (TravisEz13) [inline build.psm1:1454]:
```suggestion
    $command = "Write-Verbose `"PowerShell Version under test: `$(`$PSVersionTable.PSVersion)`" -Verbose;"
```

File: build.psm1 [hunk 2]
```diff
@@ -1450,6 +1450,7 @@ function Start-PSPester {
 
     # All concatenated commands/arguments are suffixed with the delimiter (space)
 
+    $command = "Write-Verbose `"PowerShell Version under test: `$(`$PSVersionTable.PSVersion)`" -Verbose;"
     # Disable telemetry for all startups of pwsh in tests
     $command = "`$env:POWERSHELL_TELEMETRY_OPTOUT = 'yes';"
```
Comment (TravisEz13) [inline build.psm1:1455]:
```suggestion
    $command += "`$env:POWERSHELL_TELEMETRY_OPTOUT = 'yes';"
```

File: test/tools/NamedPipeConnection/src/code/Microsoft.PowerShell.NamedPipeConnection.csproj [hunk 3]
```diff
@@ -23,7 +23,7 @@
     <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="8.0.1" />
     <PackageReference Include="System.DirectoryServices" Version="8.0.0" />
     <PackageReference Include="System.Management" Version="8.0.0" />
-    <PackageReference Include="System.Management.Automation" Version="7.4.0" />
+    <PackageReference Include="System.Management.Automation" Version="(7.4.5,)" />
```
Comment (TravisEz13) [inline test/tools/NamedPipeConnection/src/code/Microsoft.PowerShell.NamedPipeConnection.csproj:26]:
```suggestion
    <PackageReference Include="System.Management.Automation" Version="7.4.6" />
```

File: PowerShell.Common.props [hunk 4]
```diff
@@ -14,13 +14,37 @@
...
+          StandardErrorImportance="Low"
+          IgnoreExitCode="true"
+          IgnoreStandardErrorWarningFormat="true"
+          EchoOff="true"
+          >
+      <Output TaskParameter="ConsoleOutput" PropertyName="PowerShellCommitId" />
     </Exec>
+    <Message Text='PowerShellCommitId: $(PowerShellCommitId)' />
+    <PropertyGroup>
+      <FakeCommitId >547a782e838b1902c5d81b4ad958f471ded58ea8</FakeCommitId>
```
Comment (TravisEz13) [inline PowerShell.Common.props:41]:
```suggestion
      <FakeCommitId>547a782e838b1902c5d81b4ad958f471ded58ea8</FakeCommitId>
```

File: PowerShell.Common.props [hunk 5]
```diff
@@ -14,13 +14,37 @@
...
+      <PowerShellVersion Condition = "'$(PowerShellCommitId)' != ''">$(PowerShellTag)-1-g$(PowerShellCommitId)</PowerShellVersion>
+      <PowerShellVersion Condition = "'$(PowerShellCommitId)' == ''">$(PowerShellTag)-1-g$(FakeCommitId)</PowerShellVersion>
+    </PropertyGroup>
+    <!-- v7.4.5-47-gfa90c8109030f4bb489e3a020217e75f30b5bb02 -->
+     <!-- <tag>-<commits>-g<hash> -->
```
Comment (TravisEz13) [inline PowerShell.Common.props:46]:
```suggestion
```

**Review by psyberjunkie — COMMENTED** [review]
added content

**Pull request closed without merging**